tests/backup_helper.go

```diff
@@ -111,7 +111,7 @@ const (
 	storkDeploymentName                       = "stork"
 	defaultStorkDeploymentNamespace           = "kube-system"
 	UpgradeStorkImage                         = "TARGET_STORK_VERSION"
-	LatestStorkImage                          = "23.9.0"
+	LatestStorkImage                          = "24.3.0-dev"
```

Comment:
Let's update it with 24.3.0

Reply:
We always used dev builds of stork. Never use GA version for testing as the latest version

Reply:
okay you have a point. Ignore it.